src/test/ResourceFixture.cpp

```diff
@@ -24,12 +23,22 @@ ResourceFixture::ResourceFixture() {
     boost::program_options::options_description desc;
     desc.add_options()
         ("resources", boost::program_options::value<std::string>()->required(),
-         "Path where the test resources are stored");
+         "Path where the test resources are stored")
+         ("ext-path", boost::program_options::value<std::string>()->implicit_value(""),
+         "Path where extensions are stored");
 
     parse(desc);
 }
 
-std::string ResourceFixture::getResource(const std::string& name) {
+std::string ResourceFixture::getExtensionsBuildDir() const {
```

Comment:
I'm not convinced by the name of this method.
I think we don't care about this folder: the ResourceFixture should load the extensions, and don't keep this information.

Reply:
Ok, now wthere is a `ExtensionFixture` class which initializes `ExtensionProviders`